docker_registry_client/DockerRegistryClient.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 
-from docker_registry_client._BaseClient import BaseClient
```

Comment:
@jkaldon the import won't break backwards compatibility, but it does provide forward compatibility.